include/constants.hpp

```diff
@@ -87,6 +87,8 @@ constexpr auto kwdVpdInf = "com.ibm.ipzvpd.VINI";
 constexpr auto kwdCCIN = "CC";
 constexpr auto locationCodeInf = "com.ibm.ipzvpd.Location";
 constexpr auto pldmServiceName = "xyz.openbmc_project.PLDM";
+constexpr auto pimServiceName = "xyz.openbmc_project.Inventory.Manager";
```

Comment:
can you move this line after line 84, so all PIM related can stay together ?

Reply:
Why you want that? That can lead to many other requirements which are actually not required and adds no value.

Reply:
what is the difference between line 84 and 90? can we not have just one

Reply:
> Why you want that? That can lead to many other requirements which are actually not required and adds no value.

fine then.